sdk/communication/azure-communication-callingserver/src/main/java/com/azure/communication/callingserver/CallConnectionAsync.java

```diff
@@ -0,0 +1,346 @@
+        }
+    }
+
+    /**
+     * Play audio in a call.
+     *
+     * @param audioFileUri The media resource uri of the play audio request. Currently only Wave file (.wav) format
+     *                     audio prompts are supported. More specifically, the audio content in the wave file must
+     *                     be mono (single-channel), 16-bit samples with a 16,000 (16KHz) sampling rate.
+     * @param loop The flag indicating whether audio file needs to be played in loop or not.
+     * @param audioFileId An id for the media in the AudioFileUri, using which we cache the media.
+     * @param callbackUri call back uri to receive notifications.
+     * @param operationContext The value to identify context of the operation.
+     * @return the response payload for play audio operation.
+     */
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public Mono<Response<PlayAudioResult>> playAudioWithResponse(String audioFileUri,
+                                                                   boolean loop,
+                                                                   String audioFileId,
+                                                                   String callbackUri,
+                                                                   String operationContext) {
+        PlayAudioRequest playAudioRequest = new PlayAudioRequest();
+        playAudioRequest.setAudioFileUri(audioFileUri);
+        playAudioRequest.setLoop(loop);
+        playAudioRequest.setAudioFileId(audioFileId);
+        playAudioRequest.setOperationContext(operationContext);
+        playAudioRequest.setCallbackUri(callbackUri);
+        return playAudioWithResponse(playAudioRequest, Context.NONE);
+    }
+
+    /**
+     * Play audio in a call.
+     *
+     * @param audioFileUri The media resource uri of the play audio request. Currently only Wave file (.wav) format
+     *                     audio prompts are supported. More specifically, the audio content in the wave file must
+     *                     be mono (single-channel), 16-bit samples with a 16,000 (16KHz) sampling rate.
+     * @param playAudioOptions Options for play audio.
+     * @return the response payload for play audio operation.
+     */
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public Mono<Response<PlayAudioResult>> playAudioWithResponse(String audioFileUri,
+                                                                   PlayAudioOptions playAudioOptions) {
+        PlayAudioRequest playAudioRequest = PlayAudioConverter.convert(audioFileUri, playAudioOptions);
+        return playAudioWithResponse(playAudioRequest, Context.NONE);
+    }
+
+    Mono<Response<PlayAudioResult>> playAudioWithResponse(PlayAudioRequest playAudioRequest,
+                                                            Context context) {
+        try {
+            Objects.requireNonNull(playAudioRequest.getAudioFileUri(), "'audioFileUri' cannot be null.");
```

Comment:
`playAudioRequest.getAudioFileUri(),` is a server side parameter right? If so, we dont need to do checks on the client side

Reply:
only client side parameters are checked